boards/arm/rpi_pico/rpi_pico.dts

```diff
@@ -14,6 +14,7 @@
 		zephyr,sram = &sram0;
 		zephyr,flash = &flash0;
 		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
```

Comment:
please add board changes to another commit

Reply:
Okay.